Gemfile

```diff
@@ -28,6 +28,9 @@ gem 'turbolinks', '~> 5'
 
 gem 'telegram-bot', github: 'telegram-bot-rb/telegram-bot'
 #gem 'gruff'
+gem 'pronto'
+gem 'pronto-rubocop', require: false
+gem 'pronto-flay', require: false
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pronto-flay` should appear before `pronto-rubocop`.